docs/how-to.md

```diff
@@ -20,6 +20,24 @@ Now when you are browsing the web and you want to save the current page as a boo
 
 For more info see here: https://paul.kinlan.me/use-bookmarklets-on-chrome-on-android/
 
+## Using HTTP Shortcuts app on Android
+
+**Note** This allows you to share URL from any app to bookmark it to linkding
+
+Install HTTP Shortcuts from [F-Droid](https://f-droid.org/en/packages/ch.rmy.android.http_shortcuts/) or [Play Store](https://play.google.com/store/apps/details?id=ch.rmy.android.http_shortcuts).
```

Comment:
Not sure if it's a good idea to promote F-Droid at all, but IMO the Play Store link should at least be the first one as that is where people should get their apps unless they really know what they are doing.